### ISSUE TITLE
Gossip Node can return grpc Unavailable after accepting Transaction

### ISSUE THREAD

**Issue by bugbytesinc** [bugbytesinc/Hashgraph#169]
This causes the library to RETRY the transaction, resulting in a DUPLICATE_TRANSACTION error returned from the network.  This condition then propagates up the call stack as an exception.

Consider adding a check in the code to watch for this condition and return OK under this specific circumstance of BUSY followed by Grpc=Unavailable....we still need to wait for the Receipt by default which gives us a chance to sort out if the submit never happened?

**Comment by bugbytesinc** [comment]
Captured error logs:

```
2/26/2020 10:38:28 PM    Latency 0,899  Gateway: 0  After: 249710  TX: 0.0.186267@1582774707.329129800 DUPLICATE TRANSACTION vvvv
2/27/2020 3:38:27 AM  TX: { 
  "sigMap": { 
    "sigPair": [ { "pubKeyPrefix": "Kx4bDLOi", "ed25519": "aUMFXhDX/0fy5XYNOpWWGo8VJF4aybDBi6PplFtDqzBIWbhEAwiFY15AISUmvIp3Z2L/um0NxwQDH+FSsvcEAw==" } ] }, 
    "bodyBytes": "ChQKDAiz89zyBRDIvvicARIEGJuvCxICGAMYgPrp5goiAwi0ATIkMTg2MjY3OjE1ODI3NzQ3MDc6MzI5MTI5ODAwOjA6MjQ5NzEw2gEmCgQYnK8LEh5JQwa7sqdb4WNt5jqBLBR14drTtUEJFH50W/RouN4=" }
2/27/2020 3:38:27 AM  RX:(00)  { "nodeTransactionPrecheckCode": "BUSY" }
2/27/2020 3:38:27 AM  RX:(01)  "Unable to communicate with network node 0.testnet.hedera.com:50211: Status(StatusCode=Unavailable, Detail=\"GOAWAY received\")"
2/27/2020 3:38:28 AM  RX:(02)  { "nodeTransactionPrecheckCode": "DUPLICATE_TRANSACTION" }
2/26/2020 10:38:28 PM    Latency 0,899  Gateway: 0  After: 249710  TX: 0.0.186267@1582774707.329129800 DUPLICATE TRANSACTION ^^^^

2/26/2020 11:01:17 PM    Latency 0,354  Gateway: 0  After: 297068  TX: 0.0.186278@1582776077.564642200 DUPLICATE TRANSACTION vvvv
2/27/2020 4:01:17 AM  TX:  { 
  "sigMap": { 
    "sigPair": [ { "pubKeyPrefix": "Kx4bDLOi", "ed25519": "hlNKBVAq/bTQyRkr4BIMDz8U1zUZh5qtj0tonk0iNg1F2yIaCkFVGeTU/m4ewLIWED/nqLppvPNBFT+punTaAg==" } ] }, 
    "bodyBytes": "ChQKDAiN/tzyBRCYg5+NAhIEGKavCxICGAMYgPrp5goiAwi0ATIkMTg2Mjc4OjE1ODI3NzYwNzc6NTY0NjQyMjAwOjA6Mjk3MDY42gEmCgQYp68LEh4tyzulVioRiyZDtf+SrtqDm96HCjOEPbFrgsK53PU=" }
2/27/2020 4:01:17 AM  RX:(00)  "Unable to communicate with network node 0.testnet.hedera.com:50211, it may be down or not reachable."
2/27/2020 4:01:17 AM  RX:(01)  { "nodeTransactionPrecheckCode": "DUPLICATE_TRANSACTION" }
2/26/2020 11:01:17 PM    Latency 0,354  Gateway: 0  After: 297068  TX: 0.0.186278@1582776077.564642200 DUPLICATE TRANSACTION ^^^^

2/26/2020 11:01:18 PM    Latency 1,505  Gateway: 0  After: 296865  TX: 0.0.186278@1582776076.813695900 DUPLICATE TRANSACTION vvvv
2/27/2020 4:01:16 AM  TX:  { 
  "sigMap": { 
    "sigPair": [ { "pubKeyPrefix": "Kx4bDLOi", "ed25519": "hDwp+N/JIM3oh8jT9eh5ffc0adrCHC8oaX6OSLCFkNXgHoBFXN8N7RJ42t6CkOokjqSNIQGtQE3w0ZnQMBTXDg==" } ] }, 
    "bodyBytes": "ChQKDAiM/tzyBRCch4CEAxIEGKavCxICGAMYgPrp5goiAwi0ATIkMTg2Mjc4OjE1ODI3NzYwNzY6ODEzNjk1OTAwOjA6Mjk2ODY12gEmCgQYp68LEh5jamvkOLcmnE22zNUM0nQ0WnXNVR19T8/EXARTd7c=" }
2/27/2020 4:01:16 AM  RX:(00)  { "nodeTransactionPrecheckCode": "BUSY" }
2/27/2020 4:01:17 AM  RX:(01)  { "nodeTransactionPrecheckCode": "BUSY" }
2/27/2020 4:01:17 AM  RX:(02)  "Unable to communicate with network node 0.testnet.hedera.com:50211, it may be down or not reachable."
2/27/2020 4:01:18 AM  RX:(03)  { "nodeTransactionPrecheckCode": "DUPLICATE_TRANSACTION" }
2/26/2020 11:01:18 PM    Latency 1,505  Gateway: 0  After: 296865  TX: 0.0.186278@1582776076.813695900 DUPLICATE TRANSACTION ^^^^
```

**Comment by bugbytesinc** [comment]
Updating Code to query network for the Receipt in the case a gRPC `StatusCode=Unavailable` is returned.  If a receipt is returned, the logic continues with the Receipts' Header Precheck Code.  If not found, the logic will re-submit the message instead.

**Comment by bugbytesinc** [comment]
Fixed by commit 7322978c878c2f675240a0190e3e940d2d97eefc
